demos/pager/index.html

```diff
@@ -62,15 +62,15 @@ <h1>School Roster</h1>
 
             <div class="flc-pager-top">
                 <ul class="demo-pager-bar fl-force-left fl-focus">
-                    <li class="flc-pager-previous demo-pager-previous"><a href="#" title="Previous Page">&lt; Previous</a></li>
+                    <li class="flc-pager-previous demo-pager-previous"><span class="demo-pager-previous-character" aria-hidden="true">&lt;</span><a href="#"> Previous</a></li>
```

Comment:
I don't think you need the class names anymore with the characters added in directly.

Reply:
the `<` and `>` symbols should be contained within the `<a>`. If that isn't workable with the `aria-hidden="true"` we may need to find an alternative solution.

Reply:
Yes @jobara , with aria-hidden true, we can't put things that are focusable inside the span element. We'll have to go with icons as discussed after making changes in the pager component.